frameworks/AutoGluon/exec.py

```diff
@@ -68,14 +69,21 @@ def run(dataset, config):
     # Persist model in memory that is going to be predicting to get correct inference latency
     predictor.persist_models('best')
 
+    def inference_time_classification(path: str):
+        data = TabularDataset(path)
```

Comment:
`data = TabularDataset(path)` is not required, predict and predict_proba internally load as TabularDataset if you pass the string path as input.

This also allows to pass either the DataFrame or the path as input to the function, depending on what you want to measure.